### PR TITLE
dispatch_method wrapper

### LINKED ISSUE #2769: Unused task mode?
It doesn't look like TASK_METHOD is a registered *mode* anywhere.
https://github.com/radical-cybertools/radical.pilot/blob/27ba881e5258cbcd13fd7d2b2fa96ce41e434e20/src/radical/pilot/task_description.py#L12

Is this retained for historical purposes because it is out in the wild? Is it deprecated?

### PR DESCRIPTION
**Pull request by andre-merzky**
This fixes #2769

### CODE REVIEW

File: src/radical/pilot/raptor/worker.py [hunk 1]
```diff
@@ -90,6 +90,7 @@ def __init__(self, manager, rank, raptor_id):
         #     shell: execute  a shell command
         self._modes = dict()
         self.register_mode(TASK_FUNC,  self._dispatch_func)
+        self.register_mode(TASK_METH,  self._dispatch_func)
```
Comment (mtitov) [inline src/radical/pilot/raptor/worker.py:93]: should point to `self._dispatch_meth`

File: src/radical/pilot/__init__.py [hunk 2]
```diff
@@ -26,8 +26,10 @@
 from .pytask                    import PythonTask
 from .task_description          import TaskDescription
 from .task_description          import TASK_EXECUTABLE
-from .task_description          import TASK_METHOD, TASK_FUNC, TASK_FUNCTION
-from .task_description          import TASK_EXEC, TASK_EVAL, TASK_PROC, TASK_SHELL
+from .task_description          import TASK_METH, TASK_METHOD
```
[the remark this answers is not in the document]
Comment (andre-merzky) [reply src/radical/pilot/__init__.py:29]: I don't think we agreed on that just yet ;-)

File: src/radical/pilot/raptor/worker.py (hunk 1 above)
Comment (andre-merzky) [reply src/radical/pilot/raptor/worker.py:93]: Duh!

File: src/radical/pilot/__init__.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mtitov) [reply src/radical/pilot/__init__.py:29]: ah, sorry, didn't try to be sneaky, thought you agreed to make it explicit in regard to which method is provided

**Review by mtitov — APPROVED** [review]
LGTM!

**Pull request merged**